api/main_endpoints/routes/Auth.js

```diff
@@ -98,7 +98,7 @@ router.post('/login', function(req, res) {
                 // query
                 { email: user.email },
                 // update this field
-                { pagesPrinted: 0 }
+                { pagesPrinted: 0 },
```

Comment:
not sure why this file has changes, id revert it

Reply:
here I had to manually make the edit to fix one of the rebases because I had a panic

Reply:
ok just change it back